components/movementsensor/wheeledodometry/wheeledodometry.go

```diff
@@ -286,9 +286,8 @@ func (o *odometry) Readings(ctx context.Context, extra map[string]interface{}) (
 	return readings, nil
 }
 
-func (o *odometry) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error,
-) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+func (o *odometry) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
+	return &movementsensor.Accuracy{}, nil
```

Comment:
during this pr uncovered a frontend error when getAccuracy is unimplemented. this was basically my fix so that I could still see wheeled odometry on the RC card to make sure the base was spinning as I expected. this can be changed back before merging and Susmita is working on a fix